src/MoBi.UI/Views/EditContainerView.cs

```diff
@@ -197,6 +188,17 @@ public bool ContainerPropertiesEditable
          }
       }
 
+      public bool IsNameEditable
+      {
+         get => _isNameEditable;
+         set
+         {
+            _isNameEditable = value;
+            btName.Enabled = value;
+            btName.ReadOnly = !value;
```

Comment:
we still have to manage the enable and visible for `editNameButton`

And we don't keep a backing property in the case of `ContainerPropertiesEditable`. It looks like we just use the Enabled property of one of the controls.